string command
string parameter1
string parameter2
string parameter3
string parameter4